descriptors/ModuleDescriptor-template.json

```diff
@@ -2468,7 +2493,7 @@
       { "name": "DB_QUERYTIMEOUT", "value": "60000" },
       { "name": "DB_CHARSET", "value": "UTF-8" },
       { "name": "DB_MAXPOOLSIZE", "value": "5" },
-      { "name": "KAFKA_HOST", "value": "kafka" },
+      { "name": "KAFKA_HOST", "value": "localhost" },
```

Comment:
Why was this value changed?

Reply:
This is my fault-I have a separate instance of this module I've been trying to get to work with okapi-I must have changed  the descriptor on this one by mistake.  Reverting.